From d25653b75b1b543ec7fff7be805fd863fa544562 Mon Sep 17 00:00:00 2001
From: rex wang <rex.wang1@amlogic.com>
Date: Tue, 23 Jun 2020 14:42:57 +0800
Subject: [PATCH 2/3] rdk: tm2 revb ms12 support [1/]

PD#TV-20821

Problem:
CAD1.3 revb ms12 fail

Solution:
enabel ms12 supprot in dtsi

Verify:
Sky Llama CAD1.3 revb

Change-Id: I03db120be8f9bbc86a2b7f88259548c4f973f859
Signed-off-by: rex wang <rex.wang1@amlogic.com>
---
 arch/arm/boot/dts/amlogic/mesontm2_revb.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/amlogic/mesontm2_revb.dtsi b/arch/arm/boot/dts/amlogic/mesontm2_revb.dtsi
index 5930238f394e..5be735d1f694 100644
--- a/arch/arm/boot/dts/amlogic/mesontm2_revb.dtsi
+++ b/arch/arm/boot/dts/amlogic/mesontm2_revb.dtsi
@@ -568,7 +568,7 @@
 	dolby_fw: dolby_fw {
 		compatible = "amlogic, dolby_fw";
 		mem_size = <0x100000>;
-		status = "disabled";
+		status = "okay";
 	};
 
 	wdt: watchdog@0xffd0f0d0 {
-- 
2.29.0

